### ISSUE TITLE
Search: use search point and distance to filter all search results

### ISSUE THREAD

**Issue by joshi1983** [hhaccessibility/hhaccessibility.github.io#101]
#92 added a location selector using free text that saves to the user table.  Corresponding with that is a longitude, latitude, and a search radius that should apply to every search query.  Every result outside the search disk should not be returned.

**Comment by joshi1983** [comment]
This could be optimized better at some point but [app.accesslocator.com](http://app.accesslocator.com) filters out locations beyond the search radius now.  A clear way to see the effect would be to save different radius values in the profile and look at the table view of search results in Education.  If you hit "Save" in the profile with search radius at 2, you won't see any locations with a distance over 2km.

**Comment by baseer110** [comment]
We have checked the search result page feature:

1) we have sign up into the account and setup the search area distance, In our example its 5Km and we searched for Education category, the search result is under the 5Km distance as we can see in the below images.

![image](https://cloud.githubusercontent.com/assets/4041079/24053139/6beef2a6-0b0e-11e7-9e1b-2de98da087da.png)

2) The result is under the mentioned distance : 

![image](https://cloud.githubusercontent.com/assets/4041079/24053188/99181122-0b0e-11e7-98ee-4c2d8e05bdce.png)

3) This search feature gives all the result under the distance provided by the user. Its working according to the feature mentioned.
